sa/model.go

```diff
@@ -404,15 +404,15 @@ func modelToOrder(om *orderModel) (*corepb.Order, error) {
 var challTypeToUint = map[string]uint{
 	"http-01":     0,
 	"tls-sni-01":  1,
-	"dns-01":      2,
-	"tls-alpn-01": 3,
+	"tls-alpn-01": 2,
```

Comment:
Why did you swap the order of `tls-alpn` and `dns-01`?

Reply:
This is just cruft left over from the old changeset, reverted.